sfge/src/main/java/com/salesforce/rules/fls/apex/operations/FlsValidationCentral.java

```diff
@@ -157,7 +155,10 @@ public void createExpectedValidations(DmlStatementVertex vertex, SymbolProvider
                     ScopeUtil.resolveToApexValue(symbols, (ChainedVertex) childVertex);
             if (apexValue.isPresent()) {
                 validationReps.addAll(
-                        validationConverter.convertToExpectedValidations(apexValue.get()));
+                        // false for isUserModeAccessLevel because this is not an instance of
+                        // Database.method(...) where the access level can be defined with a
+                        // parameter
```

Comment:
This comment is correct but misleading.
`DmlStatementVertex` objects can specify an access level via the `as user` or `as system` syntax, but that's already handled elsewhere (see `AbstractFlsVisitor` line 96), and therefore doesn't need to be handled here explicitly.